File: android/sited/src/main/java/org/noear/sited/__ICache.java
package org.noear.sited;

/**
 * Created by yuety on 15/10/10.
 */
 interface __ICache {
    void save(String key, String data);
    __CacheBlock get(String key);
    void delete(String key);
    boolean isCached(String key);
}

File: ios/sited/sited/src/__ICache.swift
//
//  __ICache.swift
//  sited
//
//  Created by 谢月甲 on 15/10/12.
//  Copyright © 2015年 noear. All rights reserved.
//

import Foundation

internal protocol ICache{
    
    func save(key:String, data:String);
    func get(key:String)->CacheBlock;
    func delete(key:String);
    func isCached(key:String)->Bool;
}

